app/views/posts/_form.html.erb

```diff
@@ -0,0 +1,20 @@
+<% @post.errors.full_messages.each do |message| %>
+  <p><%= message %></p>
+<% end %>
+
+<%= form_for @post do |f| %>
+  <%= f.label :name %>
+  <%= f.text_field :name %>
+  <br>
+  <br>
+  <%= f.label :title %>
+  <%= f.text_field :title %>
+  <br>
+  <br>
+  <%= f.label :content %>
+  <%= f.text_area :content %>
+  <br>
+  <br>
+
+  <%= f.submit %>
+<% end %>
```

Comment:
👍 

There are sticklers about semantic html and say that `<br>` tags are stylistic and not content. But.. meh